jnigen/tool/regenerate_bindings.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
```

Comment:
nit: should we call this something with 'all' in the name? `regenerate_all_bindings.dart` or `regenerate_all.dart`